### PR TITLE
NODE-2509 Fix blinking tests

### PR DESCRIPTION
**Pull request by xrtm000**

### CODE REVIEW

**Review by ivan-mashonskiy — APPROVED** [review]

**Review by yvaskov — APPROVED** [review]

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala [hunk 1]
```diff
@@ -149,30 +146,6 @@ class FailedTransactionGrpcSuite extends GrpcBaseTransactionSuite with FailedTra
     }
   }
 
-  test("InvokeScriptTransaction: invoking asset script error in action asset propagates failed transaction") {
```
Comment (vsuharnikov) [inline node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala:152]: This definitely solves the issue :) There is the same test?

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala [hunk 2]
```diff
@@ -253,66 +226,6 @@ class FailedTransactionGrpcSuite extends GrpcBaseTransactionSuite with FailedTra
     }
   }
 
-  test("InvokeScriptTransaction: account state should not be changed after accepting failed transaction") {
```
Comment (vsuharnikov) [inline node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala:256]: Here is the same question

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/SponsorFeeTransactionGrpcSuite.scala [hunk 3]
```diff
@@ -72,7 +72,8 @@ class SponsorFeeTransactionGrpcSuite extends GrpcBaseTransactionSuite {
       sender.assetsBalance(bobAddress, Seq(sponsoredAssetId)).getOrElse(sponsoredAssetId, 0L) shouldBe bobAssetBalance + 10 * token
       sender.assetsBalance(sponsorAddress, Seq(sponsoredAssetId)).getOrElse(sponsoredAssetId, 0L) shouldBe sponsorAssetBalance + smallFee
 
-      val reward = (sender.height - minerBalanceHeight) * 600000000L
+      val height = nodes.head.waitForHeightArise()
```
Comment (vsuharnikov) [inline node-it/src/test/scala/com/wavesplatform/it/sync/grpc/SponsorFeeTransactionGrpcSuite.scala:75]: Probably, getting a balance _with confirmations_ would be better here, but I see no such method :( So let it be

File: node-it/src/test/scala/com/wavesplatform/it/sync/transactions/FailedTransactionSuite.scala [hunk 4]
```diff
@@ -165,55 +164,6 @@ class FailedTransactionSuite extends BaseTransactionSuite with CancelAfterFailur
     }
   }
 
-  test("InvokeScriptTransaction: account state should not be changed after accepting failed transaction") {
```
Comment (vsuharnikov) [inline node-it/src/test/scala/com/wavesplatform/it/sync/transactions/FailedTransactionSuite.scala:168]: Same questions here

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala (hunk 1 above)
Comment (vsuharnikov) [reply node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala:152]: Got answer: we're doing these checks in unit tests

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala (hunk 2 above)
Comment (vsuharnikov) [reply node-it/src/test/scala/com/wavesplatform/it/sync/grpc/FailedTransactionGrpcSuite.scala:256]: Got answer: we're doing these checks in unit tests

File: node-it/src/test/scala/com/wavesplatform/it/sync/transactions/FailedTransactionSuite.scala (hunk 4 above)
Comment (vsuharnikov) [reply node-it/src/test/scala/com/wavesplatform/it/sync/transactions/FailedTransactionSuite.scala:168]: Got answer: we're doing these checks in unit tests

**Review by vsuharnikov — APPROVED** [review]

**Review by phearnot — APPROVED** [review]

**Pull request merged**